eras/conway/impl/src/Cardano/Ledger/Conway/Transition.hs

```diff
@@ -27,14 +47,30 @@ import Data.Aeson (
   withObject,
   (.:),
  )
+import Data.ListMap (ListMap)
+import qualified Data.ListMap as ListMap
+import qualified Data.Map.Strict as Map
 import GHC.Generics
 import Lens.Micro
 import NoThunks.Class (NoThunks (..))
 
+class EraTransition era => ConwayEraTransition era where
+  tcDRepDelegsL ::
+    Lens'
+      (TransitionConfig era)
+      (ListMap (Credential 'Staking (EraCrypto era)) (DRep (EraCrypto era)))
+
+  tcInitialDRepsL ::
+    Lens'
+      (TransitionConfig era)
+      (ListMap (Credential 'DRepRole (EraCrypto era)) (DRepState (EraCrypto era)))
+
+  tcConwayGenesisL :: Lens' (TransitionConfig era) (ConwayGenesis (EraCrypto era))
+
 instance Crypto c => EraTransition (ConwayEra c) where
   data TransitionConfig (ConwayEra c) = ConwayTransitionConfig
-    { ctcConwayGenesis :: !(ConwayGenesis c)
-    , ctcBabbageTransitionConfig :: !(TransitionConfig (BabbageEra c))
+    { ctcConwayGenesis :: ConwayGenesis c
+    , ctcBabbageTransitionConfig :: TransitionConfig (BabbageEra c)
```

Comment:
These are fine to be strict. It is only the injection fields that need to be lazy, cause they can contain a lot of data
```suggestion
    { ctcConwayGenesis :: !(ConwayGenesis c)
    , ctcBabbageTransitionConfig :: !(TransitionConfig (BabbageEra c))
```